### ISSUE TITLE
Install `yarn` globally

### ISSUE THREAD

**Issue by netchampfaris** [frappe/bench#579]
The [new build system](https://github.com/frappe/frappe/pull/5010) requires `yarn` to be installed globally. Add a new .yml for installing `yarn` ?

**Comment by dottenbr** [comment]
We still have to `bench update --reset` because of `yarn.lock`

**Comment by dottenbr** [comment]
This is not the worst thing. Since you have introduced yarn the installation steps are no longer working. Yarn is not listed as a prerequisite anywhere. When trying to init a new bench you will get errors related to yarn, if you then install yarn through apt the init still breaks.

```
frappe@test:~$ bench init frappe-bench --frappe-path https://github.com/newmatik/frappe.git
INFO:bench.utils:virtualenv -q env -p /usr/bin/python
Already using interpreter /usr/bin/python
INFO:bench.utils:./env/bin/pip -q install --upgrade pip
INFO:bench.utils:./env/bin/pip -q install wheel
INFO:bench.utils:./env/bin/pip -q install six
INFO:bench.utils:./env/bin/pip -q install -e git+https://github.com/frappe/python-pdfkit.git#egg=pdfkit
INFO:bench.app:getting app frappe
INFO:bench.utils:git clone https://github.com/newmatik/frappe.git   --origin upstream
Cloning into 'frappe'...
remote: Counting objects: 140896, done.
remote: Total 140896 (delta 0), reused 0 (delta 0), pack-reused 140896
Receiving objects: 100% (140896/140896), 130.82 MiB | 21.22 MiB/s, done.
Resolving deltas: 100% (106102/106102), done.
Checking connectivity... done.
('installing', u'frappe')
INFO:bench.app:installing frappe
INFO:bench.utils:frappe-bench/env/bin/pip install -q  -e frappe-bench/apps/frappe --no-cache-dir
Updating node libraries...
INFO:bench.utils:yarn install
/usr/bin/env: ‘node’: No such file or directory
Traceback (most recent call last):
  File "/usr/local/bin/bench", line 9, in <module>
    load_entry_point('bench', 'console_scripts', 'bench')()
  File "/home/frappe/bench-repo/bench/cli.py", line 40, in cli
    bench_command()
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python2.7/dist-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/frappe/bench-repo/bench/commands/make.py", line 28, in init
    python 		 = python)
  File "/home/frappe/bench-repo/bench/utils.py", line 74, in init
    update_npm_packages(bench_path=path)
  File "/home/frappe/bench-repo/bench/utils.py", line 442, in update_npm_packages
    exec_cmd('yarn install', cwd=app_path)
  File "/home/frappe/bench-repo/bench/utils.py", line 142, in exec_cmd
    raise CommandFailedError(cmd)
bench.utils.CommandFailedError: yarn install
```

**Comment by jasonbrowne** [comment]
We have come Install install issues concerning yarn while performing install 


[!] Error: Unexpected token
../../node_modules/frappe-datatable/dist/frappe-datatable.css (3:0)
1: /* variables */
2: 
3: .data-table {
   ^
4: 
5:   /* styling */

created a bug for it here


https://github.com/frappe/bench/issues/581

commands and with out put can be found here 

https://oasis.sandstorm.io/shared/wqm0b0LW6VYMygQbgqWPqV576txfQkUPE2c8DqN2EZi

**Comment by jasonbrowne** [comment]
Managed to get past this error, 

I had to npm install a couple items related to the new rollbar rollup
they are listed in [package.json-lock ](https://oasis.sandstorm.io/shared/1RrpP8HtElrj8XzXw2ni8dGY2LOkIFrPtUsHmtsr0KQ)

Now we are up and running again.

**Comment by codingCoffee** [comment]
Already fixed in commit c06307e50de525f2d07b86f6dc5e3e723c1f5c33
